### ISSUE TITLE
I am getting this issue which I tried to run this app

### ISSUE THREAD

**Issue by GouthamJM** [robot-ux/wallet-core-example#3]
```
Uncaught (in promise) Error: Cannot find module './generated/core_proto'
    at webpackEmptyContext (dist|sync:2:1)
    at index.js:18:1
    at index.js:8:1
    at ./node_modules/@trustwallet/wallet-core/dist/index.js (index.js:14:1)
    at options.factory (react refresh:6:1)
    at __webpack_require__ (bootstrap:24:1)
    at fn (hot module replacement:62:1)
    at initWasm.ts:15:1
    at new Promise (<anonymous>)
    at initWasm (initWasm.ts:3:1)
```

**Comment by punithbm** [comment]
@hewigovens  getting same error for me as well

**Comment by hewigovens** [comment]
please try using 2.9.5: `pnpm install @trustwallet/wallet-core@2.9.5`, we will check latest build

**Comment by GouthamJM** [comment]
I tried the above command , webpack compiled correctly but getting error 
```
wallet-core.js:95 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'ib')
```

**Comment by punithbm** [comment]
@hewigovens  this has been resolved with version 2.9.5. Thank you
